base_report_to_printer/models/ir_actions_report.py

```diff
@@ -111,6 +113,14 @@ def print_document(self, record_ids, data=None):
             raise exceptions.Warning(
                 _('No printer configured to print this report.')
             )
+        report_file_names = [
+            safe_eval(self.print_report_name, {"object": obj, "time": time})
```

Comment:
Is this `safe_eval` the same call as standard one with the same context? Maybe you can call to the same thing for not having possible differences?